CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+Unreleased section should follow [Release Toolkit](https://github.com/newrelic/release-toolkit#readme)
```

Comment:
@alvarocabanas  here is where I was mentioned that a chuleton 🍖  would be nice to have, For a dev to know what to put in the Unreleased section.

Reply:
Yes, In the WIP PR Readme (@roobre  has commented he is going to do some modifications on the branch in the shape of commits), I added this section [user facing changelog](https://github.com/newrelic/release-toolkit/tree/add_readme#user-facing-changelog) that would cover the cheatsheet you commented.